app/controllers/surveys_controller.rb

```diff
@@ -1,4 +1,35 @@
 class SurveysController < ApplicationController
+
   def send_survey
```

Comment:
Apparently `send()` is a built-in method on a ruby object.
I tried using send and it was giving some obscure error, so i changed to `send_survey`.
The API still uses a `send` action.


reference: http://ruby-doc.org/core-2.3.2/Object.html#method-i-send